src/aoc2020/day1.clj

```diff
@@ -0,0 +1,63 @@
+(ns aoc2020.day1
+  (:require [clojure.java.io :as io]
+            [clojure.string :as str]))
+
+(defn file->seq
+  [filename]
+  (-> filename
+      (io/resource)
+      (slurp)
+      (str/split-lines)))
+
+
+(defn sum-2020->mutiply
+  "using set to find the complement"
+  [xs]
+  (->>
+    (let
+      [xs          (map #(Integer/parseInt %) xs)
+       s           (set xs)
+       complement? #(contains? s (- 2020 %))]
```

Comment:
Avoid `#()` forms for the next few weeks. Use full function forms `(fn [] ...)`. This will force you to work on naming